u8timeseries/models/standard_regression_model.py

```diff
@@ -46,7 +46,7 @@ def __init__(self,
 
     @staticmethod
     def _get_features_matrix_from_series(features: List[TimeSeries]):
-        return np.array([s.values() for s in features]).T  # (n_samples x n_features)
+        return np.array([s.univariate_values() for s in features]).T  # (n_samples x n_features)
```

Comment:
We'll also need to change this. Perhaps now we could have just one multivariate time series containing all the features.

Reply:
I will add this to my todo list for step 3. Although requiring just one big multivariate TimeSeries instance means that the user has to stack all his predictions manually beforehand.

Reply:
Perhaps we can support an arbitrary number of (possibly multivariate) series and stack them anyway.